scripts/bump-chart-version.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+parent_dir="$1"
+
+chart_file="charts/${parent_dir}/Chart.yaml"
+
+if [[ ! -f "$chart_file" ]]; then
+  echo "Chart file not found: $chart_file"
+  exit 1
+fi
+
+version=$(grep "^version:" "$chart_file" | awk '{print $2}')
```

Comment:
I know we have quite strict chart linting so this should be fine but this is ultra basic/flakey. I there a better way of parsing /editing this yaml given we're running inside a github action?

Reply:
I don't see why we would need to?

Reply:
All I'm saying is this isn't very clean and makes assumptions about formatting (for examplel that the string on the rhs isn't quoted). Now if this was json I'd just say use `jq` and I'm not sure any tools are distriubted with the github runner to do a similar job but I'm saying it's worth a look. If not this will be fine

Reply:
We could use `yq` but this is not installed by default on the github runner.  We would have to install ourselves.

Reply:
Alrigth this will do